src/libs/ajax/file-browser-providers/FileBrowserProvider.ts

```diff
@@ -3,6 +3,7 @@ import IncrementalResponse from 'src/libs/ajax/incremental-response/IncrementalR
 export interface FileBrowserFile {
   path: string;
   url: string;
+  contentType: string;
```

Comment:
wondering why this is 'string' vs text/plain?

Reply:
This is the type definition for the file. The `contentType` field can contain any MIME type. Most of the test data uses 'text/plain', but other common ones in Terra are 'application/octet-stream', 'text/tab-separated-values', etc.